reports/Pennsylvania.md

```diff
@@ -211,6 +211,23 @@ id: pa-philadelphia-11
 * https://unicornriot.ninja/2020/philadelphia-police-threaten-unicorn-riot-reporter-after-vigilante-assault/
 
 
+### Counter-Terrorism Unit detains reporter | June 23rd
+
+A reporter recounts on Twitter that she was arrested despite telling an officer she was a reporter. She states she was zip-tied, mocked and then dragged backwards down two flights of stairs to a detention area. She states that later an officer came and asked, "Are you Samantha?" When she confirmed her name, she was released.
+
+Footage shows first-person video leading up to her arrest. It includes her clear identification, with press badge, as a reporter.
+
+tags: journalist, arrest, abuse-of-power
```

Comment:
should zip-ties be a tag?

Reply:
Maybe? On the one zip ties can be real bad. On the other, it applies to basically ever arrest.

Reply:
@rainmaker @ubershmekel what do you guys think?

Reply:
Maybe a good way to go would be to add a "Volunteer Needed" issue to go through and add zip-ties to everything